requirements.txt

```diff
@@ -7,6 +7,7 @@ click==8.1.7
 distro==1.8.0
 firebase-admin==6.2.0
 Flask==3.0.0
+flask-cors==4.0.0
```

Comment:
Why do we need cross domain access

Reply:
Your understanding is correct (although, in my view, CORS is usually configured on the backend).

When we develop React applications, we typically use a front-end server (like Vite or Webpack), commonly referred to as a "development server" or "hot reload server." 

This development server can monitor changes in the project files and automatically reload the page or replace the modified modules without manually refreshing the browser. 

When multiple servers are deployed on different ports and communicate through APur understanding is correct (although, in my view, CORS is usually configured on the backend).

When we develop React applications, we typically use a front-end server (like Vite or Webpack), commonly referred to as a "development server" or "hot reload server." 

Currently, we handle CORS on the backend, but considering our plans for production deployment (where Nginx will resolve CORS issues in the production stage), we will later configure CORS on the front-end instead of the backend.

Reply:
Indeed, the primary concern is the approach to operating a development server versus a production server; they ought to be distinct. I'll implement the required modifications following the merger of this pull request.

Reply:
Our current strategy:

No backend code used at all 
- development
    We use a front-end proxy
- production
    We use nginx